Empire/Empire.cpp

```diff
@@ -206,9 +246,11 @@ namespace {
 
         projects_in_progress = 0;
         allocated_pp.clear();
-
+        allocated_stockpile_pp.clear();
+        float dummy_pp_source = 0;
+        
         //DebugLogger() << "queue size: " << queue.size();
-
+      
```

Comment:
prefer no spaces at left of blank lines, but (I think?) 6 blank spaces when the indenting is 8 is weird...